.travis.yml

```diff
@@ -106,7 +107,7 @@ jobs:
 
     -
       <<: *STANDARD_TEST_JOB
-      php: 7.4snapshot
+      php: 7.4
```

Comment:
no need for a snapshot anymore

Reply:
I think we can also add the other 'steps' for 7.4.

e.g. with deps and drivers

Reply:
There's #817 to do something like that, but there's also some difficulties which prevent builds from succeeding. I can imagine this PR isn't about that.